### PR TITLE
KTOR-3487 Drop client.get(plugin) operator

### PR DESCRIPTION
**Pull request by rsinukov**

### CODE REVIEW

File: ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/HttpClientPlugin.kt [hunk 1]
```diff
@@ -32,24 +32,18 @@ public interface HttpClientPlugin<out TConfig : Any, TPlugin : Any> {
 }
 
 /**
- * Try to get the [plugin] installed in this client. Returns `null` if the plugin was not previously installed.
+ * Returns a [plugin] installed in this client. Returns `null` if the plugin was not previously installed.
  */
 public fun <B : Any, F : Any> HttpClient.plugin(plugin: HttpClientPlugin<B, F>): F? =
```
Comment (olme04) [inline ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/HttpClientPlugin.kt:37]: Does it make sense to break it in 2.0 and do `plugin` and `pluginOrNull` naming?
same naming strategy is used in server plugins

Comment (e5l) [reply ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/HttpClientPlugin.kt:37]: +1

**Review by e5l — CHANGES_REQUESTED** [review]
let's make it the same naming as in the server

File: ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/HttpClientPlugin.kt (hunk 1 above)
Comment (rsinukov) [reply ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/HttpClientPlugin.kt]: done

**Review by e5l — APPROVED** [review]

**Pull request merged**